tests/api2/test_005_interface.py

```diff
@@ -60,8 +63,17 @@ def get_payload(ws_client):
 
     return payload, to_validate
 
+# Make sure that our initial conditions are met
+def test_001_check_ipvx(request, ws_client, get_payload):
```

Comment:
Do we want `get_payload/ws_client` here ?

Reply:
@yocalebo i think using `ws_client` should not be required anymore as we have refined `call` function now which we consume for communicating - do you agree ?